Gemfile

```diff
@@ -44,9 +44,9 @@ end
 group :development, :test do
   gem 'rspec-rails', '3.1.0'
   gem 'capybara', '~> 2.4.1'
-  gem 'cucumber-rails', '1.4.0', require: false
   gem 'pry'
   gem 'selenium-webdriver', "2.38.0"
+  gem 'byebug'
```

Comment:
The gem byebug is nice, but since there is already the gem pry, I believe that byebug could be replaced with the gem pry-nav. It does pretty much the same as byebug, but it is part of pry, so you don't have 2 tools to debug.